src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -1076,6 +1142,35 @@ def _get_latest_revision(bucket_paths):
   return None
 
 
+def _maybe_get_http_build_url(revision) -> Optional[str]:
```

Comment:
One thing to maybe note is that by converting the build URL to HTTP, I think we lose the ability to authenticate without any extra work. This may not be a problem in current chrome builds but may be a problem in the future.

Reply:
E.g. we fuzzed widevine-based builds in the past.